spec/services/adobe_pdf/accessibility_checker_spec.rb

```diff
@@ -0,0 +1,222 @@
+                         'Status' => 'Failed' }
+                     ],
+                     'Page Content' => [
+                       { 'Description' => 'All page content is tagged', 'Rule' => 'Tagged content',
+                         'Status' => 'Failed' },
+                       { 'Description' => 'All annotations are tagged', 'Rule' => 'Tagged annotations',
+                         'Status' => 'Passed' },
+                       { 'Description' => 'Tab order is consistent with structure order',
+                         'Rule' => 'Tab order', 'Status' => 'Failed' },
+                       { 'Description' => 'Reliable character encoding is provided',
+                         'Rule' => 'Character encoding', 'Status' => 'Passed' },
+                       { 'Description' => 'All multimedia objects are tagged', 'Rule' => 'Tagged multimedia',
+                         'Status' => 'Passed' },
+                       { 'Description' => 'Page will not cause screen flicker', 'Rule' => 'Screen flicker',
+                         'Status' => 'Passed' },
+                       { 'Description' => 'No inaccessible scripts', 'Rule' => 'Scripts',
+                         'Status' => 'Passed' },
+                       { 'Description' => 'Page does not require timed responses', 'Rule' => 'Timed responses',
+                         'Status' => 'Passed' },
+                       { 'Description' => 'Navigation links are not repetitive', 'Rule' => 'Navigation links',
+                         'Status' => 'Passed' }
+                     ],
+                     'Tables' => [
+                       { 'Description' => 'TR must be a child of Table, THead, TBody, or TFoot',
+                         'Rule' => 'Rows', 'Status' => 'Failed' },
+                       { 'Description' => 'TH and TD must be children of TR', 'Rule' => 'TH and TD',
+                         'Status' => 'Failed' },
+                       { 'Description' => 'Tables should have headers', 'Rule' => 'Headers',
+                         'Status' => 'Failed' },
+                       { 'Description' => 'Tables must contain the same number of columns in each row and rows in each column', 'Rule' => 'Regularity',
+                         'Status' => 'Failed' },
+                       { 'Description' => 'Tables must have a summary', 'Rule' => 'Summary',
+                         'Status' => 'Failed' }
+                     ]
+                   },
+                   'Summary' => {
+                     'Description' => 'The checker found problems which may prevent the document from being fully accessible.',
+                     'Failed' => 18,
+                     'Failed manually' => 0,
+                     'Needs manual check' => 2,
+                     'Passed' => 12,
+                     'Passed manually' => 0,
+                     'Skipped' => 0
+                   }
+                 })
```

Comment:
The request is mocked with VCR/Webmock here spec/fixtures/vcr_cassettes/AdobePdf_AccessibilityChecker/_call/happy_path/fetches_the_accessibility_report.yml, so it won't break unless that is turned off.  If we do turn that off and run a live test, it would be good to have it break so we can make changes.  I added an issue to further handle the format and storing of the report here #1637 .